src/main/java/org/jfrog/hudson/pipeline/common/executors/EditPropsExecutor.java

```diff
@@ -40,6 +40,7 @@ public void execute() throws IOException, InterruptedException {
         Boolean success = ws.act(new EditPropertiesCallable(new JenkinsBuildInfoLog(listener),
             preferredDeployer.provideUsername(build.getParent()),
             preferredDeployer.providePassword(build.getParent()),
+                preferredDeployer.provideAccessToken(build.getParent()),
```

Comment:
Can we pass a credentials object to this constructor, instead of username, password and an access token?